pkg/report/linux.go

```diff
@@ -2313,6 +2313,27 @@ var linuxOopses = append([]*oops{
 		[]*regexp.Regexp{},
 		crash.UnknownType,
 	},
+	{
+		[]byte("rust_kernel: panicked"),
+		[]oopsFormat{
+			{
+				title:  compile("rust_kernel: panicked"),
+				report: compile("rust_kernel: panicked at [^\n]*?\n(.+?)\n"),
+				fmt:    "%[1]v in %[2]v",
```

Comment:
It is hard to distinguish. All other titles have unique prefixes.
"rust_kernel panicked: %[1]v in %[2]v" for example.

I'll include this change in my PR to not forget.